docs/self-managed/reference-architecture/kubernetes.md

```diff
@@ -0,0 +1,287 @@
+  - Keycloak, an external dependency of Identity
+
+The **Orchestration cluster** should be configured to use the central Identity of the **Web Modeler and Console** namespace.
+
+## Requirements
+
+This guide focuses on a single-region application, but can be adapted for a multi-region setup once you understand the basics of a single region.
+
+For more details on multi-region configurations, especially dual-region setups, refer to the [dedicated guide](/self-managed/concepts/multi-region/dual-region.md).
+
+### Infrastructure
+
+We recommend using an officially [certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/#benefits) distribution.
+
+Camunda 8 is not tied to a specific Kubernetes version. To simplify deployment, we provide a [Helm chart](/self-managed/setup/install.md) for easy installation on Kubernetes. The latest Helm chart is typically compatible with Kubernetes' [official support cycle](https://kubernetes.io/releases/).
+
+#### Minimum cluster requirements
+
+The following are suggested minimum requirements. Sizing depends heavily on your specific use cases and workload. See [sizing your environment](/components/best-practices/architecture/sizing-your-environment.md) and [Zeebe resource planning](/self-managed/zeebe-deployment/operations/resource-planning.md) for additional resources, conduct benchmarking to determine your exact requirements.
+
+- 4 Kubernetes Nodes
+  - Modern CPU: 4 Cores
+  - Memory: 16 GiB
+- Persistent Volumes
+  - 1,000 IOPS
+  - 32 GiB size
+  - _avoid burstable disk types_
+
+#### Networking
+
+While networking is largely managed through services and Load Balancers, understanding port usage can be useful - especially if explicit whitelisting is required within a private network.
+
+- Stable, high-speed network connection
+- Configured firewall rules to allow necessary traffic:
+  - **80**: Web UI (Console, Identity, Keycloak, Operate, Optimize, Tasklist, Web Modeler)
+  - **82**: Metrics endpoint (Identity)
+  - **8080**: REST endpoint (Connectors, Keycloak, Zeebe Gateway)
+  - **8071 / 8091**: Management endpoint (Web Modeler)
+  - **8092**: Management endpoint (Optimize)
+  - **9100**: Management endpoint (Console)
+  - **9600**: Management endpoint (Operate, Tasklist, Zeebe Brokers, Zeebe Gateway)
+  - **26500**: gRPC endpoint.
+  - **26501**: Gateway-to-broker communication.
+  - **26502**: Inter-broker communication.
+- Load Balancer for distributing traffic and exposing Camunda 8 to users (if required)
+
+The exposed Kubernetes service port does not always match the target port of the components. The correct target ports can be found in the rendered Helm chart, or in the configuration pages for each component.
+
+:::note Databases
+
+Databases were not considered as those should be maintained outside and ports may differ from the upstream defaults.
+
+The defaults for the databases are:
+
+- **5432**: PostgreSQL
+- **9200 / 9300**: Elasticsearch / OpenSearch
+
+:::
```

Comment:
This note isn't terribly straightforward to me. Does this mean:
```suggestion
:::note Databases

Database ports are not included, as databases should be maintained outside Camunda, and ports may differ from the upstream defaults.

The defaults for the databases are:

- **5432**: PostgreSQL
- **9200 / 9300**: Elasticsearch/OpenSearch

:::
```

Reply:
backported it to 8.7 👍